requirements/base.txt

```diff
@@ -5,6 +5,6 @@
 #    make upgrade
 #
 django-waffle==0.12.0     # via -r requirements/base.in
-django==2.2.13            # via -c requirements/constraints.txt, -r requirements/base.in
+django==2.2.14            # via -c requirements/constraints.txt, -r requirements/base.in
```

Comment:
Does all this mean I need a task in Toggle v2 for adding a `make upgrade` job in Jenkins for edx-toggles?

Reply:
Good point. Here's a PR for it: https://github.com/edx/jenkins-job-dsl/pull/1061